kong/plugins/ai-proxy/schema.lua

```diff
@@ -0,0 +1,146 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+local request_log_compatible = {
+  "llm/v1/chat",
+  "llm/v1/completions",
+}
+
+local auth_schema = {
+  type = "record",
+  required = false,
+  fields = {
+    { header_name = { type = "string", description = "If AI model requires authentication via Authorization or API key header, specify its name here.", required = false, referenceable = true } },
+    { header_value = { type = "string", description = "Specify the full auth header value for 'header_name', for example 'Bearer key' or just 'key'.", required = false, referenceable = true } },
+    { param_name = { type = "string", description = "If AI model requires authentication via query parameter, specify its name here.", required = false, referenceable = true } },
```

Comment:
```suggestion
    { query_param_name = { type = "string", description = "If AI model requires authentication via query parameter, specify its name here.", required = false, referenceable = true } },
```

A bit more self-descriptive

Reply:
It's set to `param_name` because it can either be query or form/body, as chosen by the `param_location` config option below.